apps/extension/src/webview/index.tsx

```
@@ -0,0 +1,1709 @@
+						dispatch({
+							type: 'CLEAR_ERROR'
+						});
+					}
+					break;
+
+				case 'errorNotification':
+					console.log('⚠️ Error notification from extension:', message);
+					const errorData = message.data;
+
+					// Map error severity to toast type
+					let toastType: ToastNotification['type'] = 'error';
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)